packages/now-python/test/fixtures/28-pydantic/now.json

```
@@ -0,0 +1,14 @@
+{
+  "version": 2,
+  "builds": [{ "src": "api/[hello].py", "use": "@now/python" }],
+  "probes": [
+    {
+      "path": "/api/[hello].py",
```

I don't think this is correct. If you want to assign a query string to your function you'll probably want to invoke:  
```suggestion
      "path": "/api/Nguyễn%20Hồng%20Quân",
```

---

But I think we already have a test for the bracket syntax for path segments so why not change the file nam to `/api/hello.py` instead?

---

Thanks. I removed the "Path Segment" test.

I want to test with `POST` method.